core/templates/pages/skill-editor-page/navbar/skill-editor-navbar-breadcrumb.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="nav navbar-nav oppia-navbar-breadcrumb">
+  <span class="oppia-navbar-breadcrumb-separator"></span>
+  <span>
+    Skill Editor &gt; {{getTruncatedDescription()}}
```

Comment:
will this template work fine for the downgraded angularJS app?

Reply:
Yes! This will work.